docs/Course Wiki/CS Course Offerings/CS421.md

```diff
@@ -0,0 +1,56 @@
+# CS421
+
+CS421 (Programming Languages and Compilers) is a 3-credit hour course that counts as an Advanced Computing elective and is explicitly required for CS majors.
```

Comment:
Can you say that it's an advanced computing elective for CEs, technical elective for EEs?